src/test/java/uk/gov/justice/digital/domains/DomainExecutorTest.java

```diff
@@ -24,7 +25,7 @@
 
 import static org.junit.jupiter.api.Assertions.*;
 
-public class DomainExecutorTest extends DomainService {
+public class DomainExecutorTest extends DomainOperations {
```

Comment:
Don't extend `DomainOperations` but wire it in instead by declaring this class as a `@MicronautTest`.

Reply:
done